crates/flux-fhir-analysis/src/wf/mod.rs

```diff
@@ -362,7 +363,7 @@ impl<'a, 'tcx> Wf<'a, 'tcx> {
             }
             fhir::TyKind::OpaqueDef(_, args, _refine_args, _) => {
                 // TODO sanity check the _refine_args (though they should never fail!) but we'd need their expected sorts
-                self.check_generic_args(infcx, args)
+                self.check_generic_args(infcx, None, args)
```

Comment:
Call `check_generics_args` with the `item_id` of the `OpaqueDef` (`item_id.onwer_id.to_def_id()`)